### ISSUE TITLE
Remove rubygems dependency.

### ISSUE THREAD

**Issue by zw963** [janlelis/unicode-display_width#17]
Hi, I use this gem and rubocop,  together with a portable version ruby. (traveling ruby)

this portable ruby is very simple, even  it no rubygems, all see like work well, but
when i update rubocop and dependency, i have to change:

```rb
require 'rubygems/util'
require_relative 'constants'

module Unicode
  module DisplayWidth
    INDEX = Marshal.load(Gem::Util.gunzip(File.binread(INDEX_FILENAME)))
  end
end
```

To:
```rb
require_relative 'constants'

module Unicode
  module DisplayWidth
    INDEX = Marshal.load(File.binread(INDEX_FILENAME))
  end
end
```

And, unpack `display_width.marshal.gz` to `display_width.marshal`.
becuase in my portable ruby, no `rubygems/util`.

in my several gems (about 20+), this gem is the only gem which need rubygems,

I know rubygems is awesome, but, for portable ruby, it no use, I just need a $LOAD_PATH, it all work.

Thanks

**Comment by janlelis** [comment]
Thank you for the feedback!

I replaced the unzip functionality with a purely zlib based one on master! Can you please check if that works for you?

**Comment by zw963** [comment]
It is worked, Thanks!

**Comment by janlelis** [comment]
Awesome! It is now released with v1.3.3!
